src/schemas/json/sarif.json

```diff
@@ -97,6 +97,24 @@
           {
             "$ref": "#/definitions/hash"
           }
+        },
+
+        "properties": 
+        {
+          "description": "Key/value pairs that provide additional details about the file reference.",
+          "type": "object",
+          "default": {}
+        },
+
+        "tag": 
+        {
+          "description": "A unique set of strings that provide additional information for the associated file reference.",
+          "type": "array",
```

Comment:
Is there a way to define your own data type so you could say: `"type": "tagsList"`, and not have to repeat the `items` property?